blockbase/theme.json

```diff
@@ -347,197 +347,6 @@
 					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
 					"slug": "system-font",
 					"name": "System Font"
-				},
-				{
-					"fontFamily": "Arvo, serif",
```

Comment:
Since these fonts were `Arvo, serif` previously here, will sites continue functioning if the font now is registered with just `Arvo` in global styles?

Reply:
Yes, since the slug is just `Arvo` and thus generated CSS variable that applies the font is the same.